### PR TITLE
etcd: add -internal-dir flag

### PR DESCRIPTION
**Pull request by kelseyhightower**
etcd supports setting the path to the etcd binary directory used for
running legacy mode and upgrades.

### CODE REVIEW

File: server/usage.go [hunk 1]
```diff
@@ -15,15 +15,16 @@ Usage:
...
-  -cors=<origins>   Comma-separated list of CORS origins.
-  -v                Enabled verbose logging.
-  -vv               Enabled very verbose logging.
+  -h -help              Show this screen.
+  --version             Show version.
+  -f -force             Force a new configuration to be used.
+  -config=<path>        Path to configuration file.
+  -name=<name>          Name of this node in the etcd cluster.
+  -data-dir=<path>      Path to the data directory.
+  -internal-dir=<path>  Path to the etcd internal directory.
```
Comment (xiang90) [inline server/usage.go]: -internal-binary-dir "Path to the etcd internal binary directory"

better? 
/cc @yichengq

**Comment by xiang90** [comment]
@kelseyhightower @yichengq 

LGTM. But I hope @yichengq also agrees on this before merging it.

File: server/usage.go (hunk 1 above)
Comment (kelseyhightower) [inline server/usage.go]: Yeah.

**Comment by yichengq** [comment]
lgtm it it has been manually tested.

**Pull request merged**